tests/test_memcache.py

```diff
@@ -137,6 +138,37 @@ def test_unicode_key(self):
         value = self.mc.get(key)
         self.assertEqual(value, 5)
 
+    def test_unicode_value(self):
+        key = 'key'
+        value = six.u('Iñtërnâtiônàlizætiøn2')
+        self.mc.set(key, value)
+        cached_value = self.mc.get(key)
+        self.assertEqual(value, cached_value)
+
+    def test_binary_string(self):
+        # Binary strings should be cacheable
+        from zlib import compress, decompress
+        value = 'value_to_be_compressed'
+        compressed_value = compress(value.encode())
+
+        # Test set
+        self.mc.set('binary1', compressed_value)
+        compressed_result = self.mc.get('binary1')
+        self.assertEqual(compressed_value, compressed_result)
+        self.assertEqual(value, decompress(compressed_result).decode())
+
+        # Test add
+        self.mc.add('binary1-add', compressed_value)
```

Comment:
Isn't that more future-proof to ensure it works with the 3 methods. In that case, I'll complete `test_unicode_value()` instead.